ide-eclipse.html

```diff
@@ -0,0 +1,366 @@
+<h2>JavaFX <span class="JFX">11</span> and Eclipse</h2>
+
+<p>This section explains how to use Java <span class="JDK">11</span> and JavaFX <span class="JFX">11</span> from Eclipse. 
```

Comment:
I don't like how 11.0.1 is shown every where on the documentation. It feels that the exact same version is to be used for the sample to run.

Reply:
We can use `JDK_MAJOR` for the text, and `JDK_VERSION` where it is required? (Same for `JFX_MAJOR`/`JFX_VERSION`)